roles/openshift_logging/files/curator.yml

```diff
@@ -0,0 +1,18 @@
+# Logging example curator config file
```

Comment:
making logging dependent upon configmaps allows them to be tweaked after deployment without having to initiate every logging change with ansible.  I think it has enough benefit to fine tune your deployment that it is worth while.  I think its also reasonable to expect if you do not also move those changes back to ansible inventory that you may loose them.